configure.ac

```diff
@@ -13,6 +13,11 @@ AC_DEFINE(VMA_LIBRARY_MINOR, vma_ver_minor, [VMA Minor Version])
 AC_DEFINE(VMA_LIBRARY_REVISION, vma_ver_revision, [VMA Revision])
 AC_DEFINE(VMA_LIBRARY_RELEASE, vma_ver_release, [VMA Release])
 
+GIT_VER=`git describe --long --abbrev=40 --dirty --tags 2> /dev/null || echo ""`
+if test -n "$GIT_VER"; then GIT_VER=`echo $GIT_VER | sed -e 's/-dirty/+/' | sed s/.*-g//`; else  GIT_VER=""; fi
+
+AC_DEFINE_UNQUOTED(VMA_GIT_VERSION, "${GIT_VER}", [VMA Git Version])
+
```

Comment:
Your suggested command does not provide the same result
1. In case of no git repository an error is printed after running configure:
```
fatal: Not a git repository (or any of the parent directories): .git
```
2. There is no "dirty" flag which shows that the tree is modified. If the tree is dirty I would like to add '+' character at the end of commit number as in:
```
Git: 00ad0ebb77c93c26be3c4c4092e54c7166d1507e+
```